src/pages/follow-up.js

```diff
@@ -69,8 +69,28 @@ const FollowUP = () => {
     if (followUps.length) {
       reset(followUps[0]);
     }
+
+    Array.prototype.forEach.call(
+      document.querySelectorAll("input[type=text],textarea"),
+      function (input) {
+        input.addEventListener("keyup", function () {
+          input.value = input.value.toUpperCase();
+        });
+      }
+    );
```

Comment:
I think this code is unneccessary since we now have it in a seperate useEffect directly below